jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -47,6 +47,9 @@
 import java.lang.foreign.MemorySegment.Scope;
 import java.lang.foreign.SegmentAllocator;
 import java.lang.foreign.ValueLayout;
+/*[IF JAVA_SPEC_VERSION >= 22]*/
+import jdk.internal.access.SharedSecrets;
```

Comment:
This class does not compile in jdk21:
```
jdk21/build/normal/support/j9jcl/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java:400: error: cannot find symbol 
        nativeMH = SharedSecrets.getJavaLangInvokeAccess().nativeMethodHandle(handle);
        ^
  symbol:   variable nativeMH
  location: class InternalDowncallHandler
jdk21/build/normal/support/j9jcl/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java:400: error: cannot find symbol 
        nativeMH = SharedSecrets.getJavaLangInvokeAccess().nativeMethodHandle(handle);
                   ^
  symbol:   variable SharedSecrets
  location: class InternalDowncallHandler
jdk21/build/normal/support/j9jcl/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java:404: error: cannot find symbol 
        resolveBoundHandle();
        ^
  symbol:   method resolveBoundHandle()
  location: class InternalDowncallHandler
jdk21/build/normal/support/j9jcl/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java:405: error: cannot find symbol 
        return nativeMH;
               ^
  symbol:   variable nativeMH
  location: class InternalDowncallHandler
```

Reply:
Fixed as the related code only occurs in JDK22.